### ISSUE TITLE
Doesn't work in lua interactive mode

### ISSUE THREAD

**Issue by spectrenoir06** [hishamhm/tabular#3]
line 282:
just add "if arg and arg[0]:match("tabular.tl$") then"

**Comment by hishamhm** [comment]
@spectrenoir06 Interesting! Which Lua version did you try it on? I could not reproduce the issue:

```
Lua 5.3.3  Copyright (C) 1994-2016 Lua.org, PUC-Rio
> tabular = require"tabular"
> print(tabular({10,20,30}))
┌───┬──┐
│1 :│10│
│2 :│20│
│3 :│30│
└───┴──┘
```

I can make this addition in any case, but I'm curious about which scenarios `arg` is not available.

**Comment by spectrenoir06** [comment]
Lua 5.1.5  Copyright (C) 1994-2012 Lua.org, PUC-Rio
> tabular = require"tabular"
/usr/local/share/lua/5.1/tabular.lua:282: attempt to index global 'arg' (a nil value)
stack traceback:
	/usr/local/share/lua/5.1/tabular.lua:282: in main chunk
	[C]: in function 'require'
	stdin:1: in main chunk
	[C]: ?
> ^C

$ luajit              
LuaJIT 2.1.0-beta3 -- Copyright (C) 2005-2017 Mike Pall. http://luajit.org/
JIT: ON SSE2 SSE3 SSE4.1 BMI2 fold cse dce fwd dse narrow loop abc sink fuse
> tabular = require"tabular"
/usr/local/share/lua/5.1/tabular.lua:282: attempt to index a nil value
stack traceback:
	/usr/local/share/lua/5.1/tabular.lua:282: in main chunk
	[C]: in function 'require'
	stdin:1: in main chunk
	[C]: at 0x562ec4bd51d0
>

**Comment by spectrenoir06** [comment]
It's clearly because I use Lua 5.1 ( luajit )

**Comment by hishamhm** [comment]
Fixed in release 0.4 :) Thanks for reporting!
